app/assets/javascripts/discourse/tests/acceptance/topic-timeline-test.js

```diff
@@ -0,0 +1,336 @@
+        unpinned: null,
+        pinned: false,
+        current_post_number: 1,
+        highest_post_number: 2,
+        last_read_post_number: 0,
+        last_read_post_id: null,
+        deleted_by: {
+          id: 7,
+          username: "foo",
+          name: null,
+          avatar_template: "/letter_avatar_proxy/v4/letter/f/b19c9b/{size}.png",
+        },
+        has_deleted: false,
+        actions_summary: [
+          {
+            id: 4,
+            count: 0,
+            hidden: false,
+            can_act: true,
+          },
+          {
+            id: 8,
+            count: 0,
+            hidden: false,
+            can_act: true,
+          },
+          {
+            id: 7,
+            count: 0,
+            hidden: false,
+            can_act: true,
+          },
+        ],
+        chunk_size: 20,
+        bookmarked: false,
+        bookmarked_posts: null,
+        topic_timer: null,
+        message_bus_last_id: 5,
+        participant_count: 1,
+        show_read_indicator: false,
+        thumbnails: null,
+        slow_mode_enabled_until: null,
+        details: {
+          can_edit: true,
+          notification_level: 3,
+          notifications_reason_id: 1,
+          can_move_posts: true,
+          can_recover: true,
+          can_remove_allowed_users: true,
+          can_invite_to: true,
+          can_invite_via_email: true,
+          can_reply_as_new_topic: true,
+          can_flag_topic: true,
+          can_review_topic: true,
+          can_close_topic: true,
+          can_archive_topic: true,
+          can_split_merge_topic: true,
+          can_edit_staff_notes: true,
+          can_toggle_topic_visibility: true,
+          can_pin_unpin_topic: true,
+          can_moderate_category: true,
+          can_remove_self_id: 7,
+          participants: [
+            {
+              id: 7,
+              username: "foo",
+              name: null,
+              avatar_template:
+                "/letter_avatar_proxy/v4/letter/f/b19c9b/{size}.png",
+              post_count: 1,
+              primary_group_name: null,
+              flair_name: null,
+              flair_url: null,
+              flair_color: null,
+              flair_bg_color: null,
+              admin: true,
+              trust_level: 4,
+            },
+          ],
+          created_by: {
+            id: 7,
+            username: "foo",
+            name: null,
+            avatar_template:
+              "/letter_avatar_proxy/v4/letter/f/b19c9b/{size}.png",
+          },
+          last_poster: {
+            id: 7,
+            username: "foo",
+            name: null,
+            avatar_template:
+              "/letter_avatar_proxy/v4/letter/f/b19c9b/{size}.png",
+          },
+        },
+      });
+    });
+  });
+
+  test("Shows dates of first and last posts", async function (assert) {
+    await visit("/t/deleted-topic-with-whisper-post/129");
+    assert.equal(find(".now-date").text(), "Jul 8");
```

Comment:
```suggestion
    assert.equal(query(".now-date").innerText, "Jul 8");
```

De-jQueryfying tests 😉 

(import `query` is needed then too, and `.trim()` might be required too)